internal/core/src/exec/expression/UnaryExpr.cpp

```diff
@@ -758,53 +758,38 @@ PhyUnaryRangeFilterExpr::PreCheckOverflow() {
                 cached_overflow_res_->size() == batch_size) {
                 return cached_overflow_res_;
             }
+            auto valid = ProcessChunksForValid<T>(CanUseIndex<T>());
+            auto res_vec = std::make_shared<ColumnVector>(
+                TargetBitmap(batch_size), std::move(valid));
+            TargetBitmapView res(res_vec->GetRawData(), batch_size);
+            TargetBitmapView valid_res(res_vec->GetValidRawData(), batch_size);
+            cached_overflow_res_ = res_vec;
```

Comment:
seems cached_overflow_res_ no longer useful, bcz valid_data may be different in different batches